src/objects/zcl_abapgit_object_fugr.clas.abap

```diff
@@ -1204,12 +1204,16 @@ CLASS zcl_abapgit_object_fugr IMPLEMENTATION.
   METHOD zif_abapgit_object~deserialize.
 
     DATA: lv_program_name TYPE syrepid,
+          lv_abap_version TYPE trdir-uccheck,
           lt_functions    TYPE ty_function_tt,
           lt_dynpros      TYPE ty_dynpro_tt,
           ls_cua          TYPE ty_cua.
 
+    lv_abap_version = get_abap_version( io_xml ).
+
     deserialize_xml(
       ii_xml       = io_xml
+      iv_version   = lv_abap_version
```

Comment:
Is there a reason why you called the parameter just "version"? I even saw in your some other PR (still draft) you renamed "version" to "state",

When I see a parameter called "version" related to API's for ABAP repository objects, I assume it's the flag for the active/inactive version. If "version" is the ABAP Language version, this might be misleading.

Therefore, I would add something to the parameter name to indicate it is the ABAP Language version (e.g., `iv_abap_version`, `iv_abap_language_version`, ...)

What do you think?

Reply:
I tried to stick with `version` (`iv_version`) when it's related to `INSERT/READ REPORT` ie. the classic "Unicode" flag for source code. It's also shorter and avoids having to realign parameters :wink:.

Best to see the final suggested state in #6476

Reply:
> I tried to stick with version (iv_version) when it's related to INSERT/READ REPORT 

OK, I see. I still would prefer to make it more explicit to differentiate. For sure, this is up to you. :)
Feel free to set this as resolved.